index.html

```diff
@@ -1,579 +1,998 @@
+        <a href="https://www.w3.org/TR/did-core/#did-syntax">DID Syntax</a>.
+      </p>
+      <p>
+        Avoid allowing users to control the structure of the DID Unique
+        Suffix, also referred to as the <code>id-char</code> in the ABNF. This
+        can be accomplished by coupling the create operation canonical
+        representation to the the <code>id-char</code>. It can also be
+        accomplished by leveraging sources of entropy.
+      </p>
+
+      <p>
+        Avoid vanity DIDs, for example,
+        <code>did:example:deadbeef1f236...</code>, the structure of the
+        identifier should not be relied on for anything other than collision
+        resistance. Be especially careful of trusting a relationship between a
+        did subject and did controller without relying on a cryptographically
+        secure verification relationship for
+        <a href="authentication">authentication</a>.
+        Anyone can claim a DID, but only someone who controlls the associated
+        keys can authenticate as it.
+        For example `did:example:wallmart` should not be assumed to belong to
+        Wallmart.
+      </p>
+
+      <p>
+        Depending on the nature of the
+        <a href="#verifiable-data-registry">VDR</a>, publishing
+        <code>type</code> or <code>kind</code> might be advantageous for
+        decentralized discovery or mass surveilance. Implementers are not to
+        rely on conformance to specification text while implementing
+        decentralized identifiers.
+      </p>
+    </section>
+
+    <section class="informative">
+      <h2>Verifiable Data Registry</h2>
+
+      <p class="note">
+        See [[DID-CORE]] for normative requirements associated with developing
+        <a href="https://www.w3.org/TR/did-core/#dfn-verifiable-data-registry">Verifiable Data Registry</a>.
+      </p>
+
+      <p>
+        The <a href="#verifiable-data-registry">VDR</a> is the system to which
+        DID Operations such as Create, Resolve, Update and Deactivate are
+        applied.
+      </p>
+
+      <p>
+        Typically a globally unique tamper evident immutable log is used to
+        produce a verifiable data registry.
+      </p>
+
+      <p>
+        Blockchains or distributed ledgers are popular choices for building
+        verifiable data registries, but there are other solutions which yield
+        unique security pros and cons.
+      </p>
+
+      <p>
+        Some DID methods rely on centralized trusted service providers and
+        merkle proofs, to provide tamper evident logs without the costs
+        associated with distributeted consensus.
+      </p>
+
+      <p>
+        Some DID methods rely on multiple registries, that may not be globally
+        available.
+      </p>
+
+      <p>
+        Some DID methods might rely on legacy security infrastructure, such as
+        TLS or TOR.
+      </p>
+
+      <p>
+        Few DID methods support portability accross verifiable data
```

Comment:
For methods that employ multiple backing registries, it is sensible to enable portability between those registries. There are certainly design considerations for DID methods that have this capability.

The change in backing registry can be more strongly signalled by a change in the DID string and metadata such as canonical ID.